Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -19,7 +62,23 @@ public static void Sort(int[] array)
         /// <param name="array">Input array in a random order.</param>
         public static void Reverse(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0)
+            {
+                return;
+            }
+
+            int[] newArray = new int[array.Length];
+
+            for (int i = 0; i < array.Length; i++)
+            {
+                newArray[i] = array[array.Length - 1 - i];
+            }
+
+            for (int i = 0; i < newArray.Length; i++)
+            {
+                array[i] = newArray[i];
+            }
```

Comment:
This looks very inefficient. You can do this better!
You don't even need newArray! :)

Reply:
this is now half the length nice one thanks.